code/game/machinery/computer/pod.dm

```diff
@@ -205,8 +205,8 @@
 		if(href_list["dstele"])
 			var/choices = list(0)
 			var/list/reachable_levels = levels_by_trait(REACHABLE)
-			for(var/datum/space_level/S in reachable_levels)
```

Comment:
Why did you change this? levels_by_trait returns a list of space_level datums.

Reply:
See my above comment, and note what the code actually does.

It returns a list of `/datum/space_level`, adds their `zpos` (numerical z value) to a list, then promptly says "who cares" and discards the list. 

Anywhere and everywhere `levels_by_trait` is utilized, it always returns a list of space levels, then ends up being used for their z-value anyway.

No point in having extra code outside of a proc if this is what it's actually being used for literally 100% of the time

Reply:
o i c